src/legacy/core_plugins/data/public/actions/filters/create_filters_from_event.ts

```diff
@@ -17,21 +17,31 @@
  * under the License.
  */
 
-import { esFilters } from '../../../../../../plugins/data/public';
+import { KibanaDatatable } from '../../../../../../plugins/expressions/public';
+import { esFilters, Filter } from '../../../../../../plugins/data/public';
 import { deserializeAggConfig } from '../../search/expressions/utils';
 // eslint-disable-next-line @kbn/eslint/no-restricted-paths
 import { getIndexPatterns } from '../../../../../../plugins/data/public/services';
 
+type Table = Pick<KibanaDatatable, 'rows' | 'columns'>;
```

Comment:
I think `Table` is a very common name for this case. Can we rename it to a more specific name for that case